testsuite/domain/src/test/resources/host-configs/host-primary-jvm-properties.xml

```diff
@@ -0,0 +1,215 @@
+    </paths>
+
+    <management>
+        <audit-log>
+            <formatters>
+                <json-formatter name="json-formatter"/>
+            </formatters>
+            <handlers>
+                <file-handler name="host-file" formatter="json-formatter" path="audit-log.log" relative-to="jboss.domain.data.dir"/>
+                <file-handler name="server-file" formatter="json-formatter" path="audit-log.log" relative-to="jboss.server.data.dir"/>
+            </handlers>
+            <logger log-boot="true" log-read-only="true" enabled="false">
+                <handlers>
+                    <handler name="host-file"/>
+                </handlers>
+            </logger>
+            <server-logger log-boot="true" log-read-only="true" enabled="false">
+                <handlers>
+                    <handler name="server-file"/>
+                </handlers>
+            </server-logger>
+        </audit-log>
+        <management-interfaces>
+            <native-interface sasl-authentication-factory="management-sasl-authentication">
+                <socket interface="management" port="9999"/>
+            </native-interface>
+            <http-interface http-authentication-factory="management-http-authentication">
+                <http-upgrade enabled="true" sasl-authentication-factory="management-sasl-authentication"/>
+                <socket interface="management" port="${jboss.management.http.port:9990}"/>
+            </http-interface>
+        </management-interfaces>
+    </management>
+
+    <domain-controller>
+        <local/>
+    </domain-controller>
+
+    <interfaces>
+        <interface name="management">
+            <inet-address value="${jboss.test.host.primary.address}"/>
+        </interface>
+        <!-- AS7-4177 specify the "public" interface in each server to test that handling -->
+    </interfaces>
+
+    <jvms>
+        <jvm name="default">
+            <heap size="64m" max-size="128m"/>
+            <jvm-options>
+                <option value="-ea"/>
+                <option value="-Dtest.jboss.server.base.dir=${jboss.server.base.dir}"/>
+                <option value="-Dtest.jboss.server.data.dir=${jboss.server.data.dir}"/>
+                <option value="-Dtest.jboss.server.log.dir=${jboss.server.log.dir}"/>
+                <option value="-Dtest.jboss.server.temp.dir=${jboss.server.temp.dir}"/>
+                <option value="-Dtest.jboss.server.name=${jboss.server.name}"/>
+            </jvm-options>
+            <environment-variables>
+                <variable name="DOMAIN_TEST_JVM" value="jvm"/>
+            </environment-variables>
+            <!--
+            <jvm-options>
+                <option value="-Xdebug"/>
+                <option value="-Xnoagent"/>
+                <option value="-Djava.compiler=NONE"/>
+                <option value="-agentlib:jdwp=transport=dt_socket,address=5005,server=y,suspend=y"/>
+
+            </jvm-options>
+            -->
```

Comment:
Would you mind removing this comment from the file? It will tend to keep it there and even copy pasted without any good reason

Reply:
Sure, I have just removed and squashed the commit. Thanks